### ISSUE TITLE
Accessibility - Sign In Screen - Incorrect DOL Seal ALT Text

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OCIO-DOLSafety-Android#7]
The ALT text for the DOL seal should be “United States Department of Labor seal” and not “DOL logo”.
